jbpm-task-assigning/scripts/build-kie-server.sh

```diff
@@ -0,0 +1,75 @@
+# Development script for quickly generating a kie-server.war file that contains all the task assigning
+# related jars until this code is moved to the final location in kie-sever-parent project structure and is
```

Comment:
Should be removed before merging. I know it seems obvious but the PR is so huge this could slip through. This comment will make it stick out.

Reply:
yes, this won't go into the kie-server-parent structure